src/main/java/com/mjd/bank/controllers/AccountController.java

```diff
@@ -25,4 +33,8 @@ public SimpleMessageResponse deposit(@RequestHeader(name = "owner_id") Long owne
     return accountService.deposit(ownerId, depositRequest.getAccountNumber(), depositRequest.getAmount());
   }
 
+  @PostMapping ("/create")
+  public SimpleMessageResponse create(@RequestHeader(name = "owner_id") Long ownerId, @RequestBody CreateRequest type) {
+    return accountService.create(ownerId, AccountType.getAccountType(type.getType().toUpperCase()));
+  }
```

Comment:
El servicio debería recibir el CreateRequest, no solo el type. Y dentro del service luego se hace el `AccountType.getAccountType(type.getType().toUpperCase())`
```suggestion
  @PostMapping ("/create")
  public SimpleMessageResponse create(@RequestHeader(name = "owner_id") Long ownerId, @RequestBody CreateRequest creationRequest) {
    return accountService.create(ownerId, creationRequest);
  }
```

Reply:
Listo